dynamodb/README.md

```diff
@@ -0,0 +1,136 @@
+# HarbourBridge: Turnkey DynamoDB-to-Spanner Evaluation
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL or DynamoDB database. This
+README provides details of the tool's DynamoDB capabilities. For general
+HarbourBridge information see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+
+## Example DynamoDB Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+Before running HarbourBridge, make sure that you have
+[set up your AWS credentials/region](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+correctly. HarbourBridge accesses your DynamoDB database via the aws go sdk
+package.
+
+To use the tool on a DynamoDB database (it will migrate all tables),
+run
+
+```sh
+harbourbridge -driver=dynamodb
+```
+
+Due the schemaless nature of DynamoDB, the tool infers the schema based on a
+certain amount of sampled data, by default, 100000 rows. If a table has more
+rows than the default value, we only use 100000 rows for estimating the schema.
+You can change this value via the flag `schema-sample-size`. For example,
+
+```sh
+harbourbridge -driver=dynamodb -schema-sample-size=500000
+```
+
+## Schema Conversion
+
+The HarbourBridge tool maps DynamoDB types to Spanner types as follows:
+
+| PostgreSQL Type    | Spanner Type               | Notes                                     |
+| ------------------ | -------------------------- | ----------------------------------------- |
+| `Number`           | `NUMERIC` or `STRING`      | defaults to NUMERIC, otherwise, STRING    |
+| `String`           | `STRING`                   |                                           |
+| `Boolean`          | `BOOL`                     |                                           |
+| `Binary`           | `BYTES`                    |                                           |
+| `Null`             | A nullable column type     |                                           |
+| `List`             | `STRING`                   | json encoding                             |
+| `Map`              | `STRING`                   | json encoding                             |
+| `StringSet`        | `ARRAY<STRING>`            |                                           |
+| `NumberSet`        | `ARRAY<NUMERIC or STRING>` |                                           |
+| `BinarySet`        | `ARRAY<BYTES>`             |                                           |
+
+We discuss these, as well as other limits and notes on schema conversion, in the
+following sections.
+
+### Schema Inference
+
+DynamoDB is schemaless database: other than a primary index and optional
```

Comment:
Missing "a" i.e. "DynamoDB is a schemaless"

I think this was probably my typo -- sorry!